CZ/pyranda_rayleigh_taylor/rayleigh_taylor_lc_build_venv_sim.yaml

```diff
@@ -0,0 +1,120 @@
+description:
+    name: rayleigh_taylor_pyranda
+    description: A study for running Rayleigh-Taylor mixing simulations using pyranda.
+    
+env:
+    variables:
+        OUTPUT_PATH: ./RT_STUDIES
+        DIM: "2D"                # Dimensionality of the model: 2D or 3D
+        VIZ_FREQ_CYCLE: 200      # Scalar data output interval [cycles]
+        DUMP_FREQ_CYCLE: 1000    # Mesh data output interval [cycles]
+        STOP_WIDTH: 0.75         # Stop when mixing layer reaches this fraction of domain 
+        NPTS: 64
+        RANVEL: 0.0
+        VENV_TARGET: $(OUTPUT_PATH)/PYRANDA_ENV
+        #PYTHON: $(OUTPUT_PATH)/PYRANDA_ENV/bin/python # NOTE: apparently can't use other env vars here :/
+        PYTHON: /usr/workspace/pyranda/bin/pyranda3.9
+    # labels:
+    #     PYTHON: $(VENV_TARGET)/bin/python
+    # NOTE: add better error message to empty 'labels' mapping in maestro
+    # labels:
+        # NOTE: wire up input deck to accept this? or just leave it generic..
+        # MIXCURVE_FILE: mix_width_at_$(ATWOOD.label).dat
+
+    dependencies:
+      paths:
+        - name: RAYLEIGH_TAYLOR_MODEL
+          path: rayleigh_taylor.py # Implicitly at $(SPECROOT)/rayleigh_taylor.py
+      git:
+        - name: PYRANDA
+          path: $(OUTPUT_PATH)
+          # url: git@github.com:LLNL/pyranda.git
+          url: https://github.com/LLNL/pyranda.git
+
+batch:
+    type        : slurm
+    host        : rzwhippet
+    bank        : wbronze
+    queue       : pdebug
+
+
+study:
+    # - name: make-env-lc
```

Comment:
want to uncomment these make-env steps given they're the 'build_venv' variants of the studies?